### ISSUE TITLE
Only 5 graphs generating

### ISSUE THREAD

**Issue by weeelee** [baieric/chatstats#6]
Getting the following after running on macOS:

Traceback (most recent call last):
  File "chatstats.py", line 207, in <module>
    main(sys.argv)
  File "chatstats.py", line 191, in main
    grapher.graph(messages, output_folder, parent_folder)
  File "/Users/User/Downloads/chatstats-master/grapher.py", line 241, in graph
    img =  plt.imread(file)
  File "/Users/User/Library/Python/3.8/lib/python/site-packages/matplotlib/pyplot.py", line 2246, in imread
    return matplotlib.image.imread(fname, format)
  File "/Users/User/Library/Python/3.8/lib/python/site-packages/matplotlib/image.py", line 1490, in imread
    with img_open(fname) as image:
  File "/Users/User/Library/Python/3.8/lib/python/site-packages/PIL/ImageFile.py", line 105, in __init__
    self.fp = open(fp, "rb")

It's only generating the following graphs:
Days with the most messages
Messages by hour of day
Messages by trimester
Messages by weekday
Number of messages sent

Please let me know what other relevant info I can provide. Thanks!

**Comment by baieric** [comment]
ChatStats is likely using an out-of-date version of Facebook's Download Your Information API... Unfortunately I do not actively maintain this codebase to keep it up to date whenever Facebook makes changes.

It looks like this error is coming from `TopStickersMessagesGraph`. You could either fix this error by updating the code to point to the proper path for wherever the sticker images are now stored in the file structure of your Facebook files. Or, if that sounds too complicated, go to the `grapher.py` file and delete the usage of `TopStickersMessagesGraph()` on line 552, maybe that will let you proceed to the other graphs instead of crashing there?

I haven't made the time to test this out myself, sorry! Happy to follow up if that doesn't work. Also, I'm curious how you made your way here? This is a pretty old project haha.

**Comment by weeelee** [comment]
Deleting the usage worked. I'll update the code to the proper path sometime later and update this.  

I saw a reddit post with someone doing this for WhatsApp so I looked into people doing it for Messenger. I've fiddled with a couple with varying results - the emojis seem to always be a disparity. I suppose they're harder to filter out maybe when they're used consecutively without spaces? At least that's what I experience in my manual experience of playing around with the raw data. I appreciate your work on this though, it's the most fleshed out by far. I wish I could fix the combine.json to go in numerical order though. 

Thanks!
